google-cloud-storage/src/test/java/com/google/cloud/storage/conformance/retry/ITRetryConformanceTest.java

```diff
@@ -276,69 +283,76 @@ private List<RetryTestCase> generateTestCases(
         RpcMethodMappings rpcMethodMappings, List<RetryTest> retryTests) {
 
       List<RetryTestCase> testCases = new ArrayList<>();
-      for (RetryTest testCase : retryTests) {
-        for (InstructionList instructionList : testCase.getCasesList()) {
-          for (Method method : testCase.getMethodsList()) {
-            String methodName = method.getName();
-            RpcMethod key = RpcMethod.storage.lookup.get(methodName);
-            assertNotNull(
-                String.format("Unable to resolve RpcMethod for value '%s'", methodName), key);
-            // get all RpcMethodMappings which are defined for key
-            List<RpcMethodMapping> mappings =
-                rpcMethodMappings.get(key).stream()
-                    .sorted(Comparator.comparingInt(RpcMethodMapping::getMappingId))
-                    .collect(Collectors.toList());
-            // if we don't have any mappings defined for the provide key, generate a case that when
-            // run reports an ignored test. This is done for the sake of completeness and to be
-            // aware of a lack of mapping.
-            if (mappings.isEmpty() && CIUtils.verbose()) {
-              TestRetryConformance testRetryConformance =
-                  new TestRetryConformance(
-                      projectId,
-                      host,
-                      testCase.getId(),
-                      method,
-                      instructionList,
-                      testCase.getPreconditionProvided(),
-                      false);
-              if (testAllowFilter.test(key, testRetryConformance)) {
-                testCases.add(
-                    new RetryTestCase(testRetryConformance, RpcMethodMapping.notImplemented(key)));
-              }
-            } else {
-              for (RpcMethodMapping mapping : mappings) {
+      Transport[] values = Transport.values();
+      for (Transport transport : values) {
```

Comment:
The change here is the introduction of the transport outer loop.

The existing loops had their indentation increased with the new outer loop. Github is doing a not so great job of illustrating that here.